src/main/resources/application-local.yaml

```diff
@@ -25,6 +25,9 @@ darts:
     enabled: ${TESTING_SUPPORT_ENDPOINTS_ENABLED:true}
   redis:
     ssl-enabled: false
+  storage:
+    arm:
+      token-base-url: ${ARM_TOKEN_BASE_URL:localhost:4551}
```

Comment:
Is this missing the protocol prefix - `http://`?

Reply:
I don't think the protocol is required here, have tested locally with no issues.